TWLight/applications/models.py

```diff
@@ -322,18 +317,20 @@ def get_user_instructions(self):
                 resource.authorization_method == Partner.PROXY
                 and resource.get_access_url
             ):
+                # fmt: off
                 # Translators: This text goes into account approval emails in the case that we need to send the user a programmatically generated link to a resource.
-                user_instructions = _(
-                    "Access URL: {access_url}".format(
+                user_instructions = _("Access URL: {access_url}".format(
                         access_url=resource.get_access_url
                     )
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string